### ISSUE TITLE
automated releases

### ISSUE THREAD

**Issue by zeke** [electron-userland/electron-prebuilt-compile#55]
Let's make it so every time a new version of `electron` is published, this module updates itself with little or no human intervention.

@paulcbetts what does the current release process look like?

**Comment by jacobq** [comment]
In the meantime, please release for [`2.0.0-beta.2`](https://electronjs.org/releases#2.0.0-beta.2) :pray:

**Comment by jacobq** [comment]
Though obviously I've never done it, the [commit history](https://github.com/electron-userland/electron-prebuilt-compile/compare/64f62104dd5f531bdb515eb45f5f4fb0869e72fd...5564372bb6f5a306ccff369314e73b8905d77788) suggests that the process goes something like this:

1. Update `version` & `dependencies.electron` in `package.json` to the new version number.
2. (optional) Commit / push
3. (optional) Tag
4. `npm publish`

**Comment by jacobq** [comment]
Is there any way that I could put a "bounty" on this? I'd be happy to PayPal someone $20 if it would help get this landed.

**Comment by spasma** [comment]
@jacobq I've just created a simple update script in my fork @ https://github.com/spasma/electron-prebuilt-compile

In the meantime you can use it by using 
`"electron-prebuilt-compile": "git+https://git@github.com/spasma/electron-prebuilt-compile.git"
in your package.json.

I understand you don't want to use it for security reasons, but I'm using it for my own projects currently.. 
It's currently built for the current latest 4.0.5

**Comment by jacobq** [comment]
@spasma thanks for the contribution. Actually, I would love to use it, but because of the way my build infrastructure works (uses `electron-forge`) I cannot easily specify an alternative for this package. `electron-forge` requires that `electron-prebuilt-compile` be specified as an "exact version", i.e. not a URL/branch/commitish. In the past I have forked `electron-forge` to modify this behavior and made my own release, like you did, but it is a pain in the neck and feels so hacky. I would much rather like to help get a real solution implemented upstream / on the main line.

**Comment by spasma** [comment]
@jacobq I totally agree on that it's too hacky for production.. It should definitely be automated by the devs.. They already have some npm scripts that will make it quite easy as you mentioned before.. I don't understand, `electron-forge` is also part of the `electron-userland` group..

**Comment by jacobq** [comment]
@zeke and @paulcbetts Is there anything I can do to help get this landed? It looks like we could do this using two complementary bots:

1. [`renovatebot`](https://github.com/renovatebot) or [`dependabot`](https://dependabot.com/#how-it-works) to automatically make commits when new electron versions come out (since we have `electron` in `package.json`'s `dependencies`)
2. [`semantic-release`](https://github.com/semantic-release/semantic-release#how-does-it-work) automatically bumps `version` and publishes new release after updating
